src/public/js/EphemeralMessage/index.js

```diff
@@ -1,15 +1,21 @@
 import store from '@js/store';
 import {renderEphemeralDetails} from '@js/EphemeralMessageRenderer';
 import isPlainObject from 'lodash/isPlainObject';
-import {addSystemMessage} from '@js/Togethernet/systemMessage';
+import {addSystemConfirmMessage} from '@js/Togethernet/systemMessage';
+import {addSystemNotifyMessage} from '@js/Togethernet/systemMessage';
```

Comment:
if you are importing 2 things from the same module, you can list them and separate them with a comma, so -

```suggestion
import {addSystemConfirmMessage , addSystemNotifyMessage} from '@js/Togethernet/systemMessage';
```

Reply:
Thanks, resolved in this [commit](https://github.com/together-support/togethernet/pull/57/commits/1d2e73bbeb7370c3b9c138a318d4a02c38fe3e82).